apps/protailwind/src/video/lesson-list.tsx

```diff
@@ -221,43 +224,42 @@ export const LessonList: React.FC<{
             Section Resources
           </p>
           <ul className="flex flex-col divide-y divide-gray-800/0">
-            {section?.resources?.map(
-              (resource: SanityDocument, resourceIdx: number) => {
-                return (
-                  <li
-                    key={resource.slug?.current || resource.slug}
-                    className="pt-2"
+            {section?.resources?.map((resource: any, resourceIdx: number) => {
+              // resources can have a lot of shapes here so the type gets weird
+              return (
+                <li
+                  key={resource.slug?.current || resource.slug}
+                  className="pt-2"
+                >
+                  <Link
+                    href={resource.url}
+                    passHref
+                    className="flex items-center px-4 py-2 font-semibold leading-tight hover:bg-gray-800"
+                    onClick={() => {
+                      track('clicked link resource in navigator', {
+                        module: module.slug.current,
+                        ...(section && {section: section.slug}),
+                        lesson: router.asPath.split('/').pop(),
+                        moduleType: module.moduleType,
+                        resource: resource.slug,
+                      })
+                    }}
+                    target="_blank"
```

Comment:
here we have something like a `UrlResource` but all resources won't have a URL 🤔